README.md

```diff
@@ -57,6 +57,8 @@ The following tools are required for development:
         brew install postgresql@9.6
         brew link postgresql@9.6 --force
 
+      postgresql@9.6 is used in prod, since it is not supported any more, installing the latest version for development should work.
```

Comment:
If it's no longer supported, should we be making plans to upgrade it in production?  Definitely worth bringing up with the rest of the team.

Reply:
Yes, I was discussing this with @emrojo. We need to inform the DBA about it I guess.